src/types.ts

```diff
@@ -497,21 +497,20 @@ export interface DbPayment {
   totalFees: number;
 }
 
+export interface Payment extends Omit<DbPayment, "id"> {
+  id: number;
+}
+
 export interface DbPermission {
   id?: number;
   createdAt: string;
   allowanceId: number;
   host: string;
-  method: string;
+  method: "signMessage";
```

Comment:
why is this a fixed value? there are different string values in that.

Reply:
Yes, see comment above, should be types.

Reply:
then why not do it directly? list them all of leave a "string" 
otherwise it will be missed or just takes ages.

Reply:
Wanted your thoughts on this first. Will adjust.